scripts/validate_docstrings.py

```diff
@@ -103,33 +103,33 @@ def get_api_items(api_doc_fd):
     previous_line = current_section = current_subsection = ""
     position = None
     for line in api_doc_fd:
-        line = line.strip()
-        if len(line) == len(previous_line):
-            if set(line) == set("-"):
+        line_stripped = line.strip()
+        if len(line_stripped) == len(previous_line):
+            if set(line_stripped) == set("-"):
                 current_section = previous_line
                 continue
-            if set(line) == set("~"):
+            if set(line_stripped) == set("~"):
                 current_subsection = previous_line
                 continue
 
-        if line.startswith(".. currentmodule::"):
-            current_module = line.replace(".. currentmodule::", "").strip()
+        if line_stripped.startswith(".. currentmodule::"):
+            current_module = line_stripped.replace(".. currentmodule::", "").strip()
             continue
 
-        if line == ".. autosummary::":
+        if line_stripped == ".. autosummary::":
             position = "autosummary"
             continue
 
         if position == "autosummary":
-            if line == "":
+            if line_stripped == "":
                 position = "items"
                 continue
 
         if position == "items":
-            if line == "":
+            if line_stripped == "":
                 position = None
                 continue
-            item = line.strip()
+            item = line_stripped.strip()
```

Comment:
lol, the improved name you've chosen actually makes it clear that this line is probably unnecessary 😄 if you wanted to clean this up in a follow-up, that'd be welcome